doc/source/servers/xgboost.md

```diff
@@ -4,10 +4,13 @@ If you have a trained XGBoost model saved you can deploy it simply using Seldon'
 
 Prequisites:
 
-  * Use xgboost v0.82
```

Comment:
Might be worth linking to https://xgboost.readthedocs.io/en/latest/tutorials/saving_model.html#a-note-on-backward-compatibility-of-models-and-memory-snapshots

XGBoost is backwards compartible post 1.0.

Not sure why we would stick with 0.82 when latest is 1.2.0 and 1.2.0 can load pre 1.0 models (with a warning).